extensions/storage-system/pkg/config/config.go

```diff
@@ -19,16 +19,14 @@ type Config struct {
 
 	TokenManager     *TokenManager `yaml:"token_manager"`
 	Reva             *Reva         `yaml:"reva"`
-	SystemUserID     string        `yaml:"system_user_id" env:"OCIS_SYSTEM_USER_ID"`
-	SystemUserAPIKey string        `yaml:"system_user_api_key" env:"OCIS_SYSTEM_USER_API_KEY"`
+	SystemUserID     string        `yaml:"system_user_id" env:"OCIS_SYSTEM_USER_ID" desc:"ID of the oCIS storage-system system user."`
```

Comment:
Where does an admin get the ID from, how is it defined?

Reply:
The admin configures the ID here. It's any string you can image. Preferrably an UUIDv4. All services using the storage-system storage, need to know that uuid

Reply:
Similar suggestion as above, to be added to the current string:
`Admins can set the ID for the storage-system system user in this config option manually which is then used to reference the user. Any reasonable long string is possibe, perferably this would be an UUIDv4 format.`